ops/relation.py

```diff
@@ -295,54 +290,29 @@ class ConsumerBase(Object):
 
         multi: a Boolean flag that indicates if the :class:`ConsumerBase` derived
```

Comment:
I would have expected we could get rid of this `multi` argument entirely. I mean `ops.relation` always works in `multi` mode, i.e. always allowing the relation between providers and consumers to be many to many. If charms want to impose such limits, they can do so with `limit` in `metadata.yaml` which is now exposed to charms through a recent merged PR by Michele.